test/validators.test.js

```diff
@@ -13865,18 +13865,30 @@ describe('Validators', () => {
       validator: 'isVAT',
       args: ['CH'],
       valid: [
-        'CH123456TVA',
-        '123456TVA',
-        'CH123456789MWST',
-        '123456789MWST',
-        'CH123.456IVA',
-        '123.456IVA',
-        'CH123.456.789TVA',
-        '123.456.789TVA',
-      ],
-      invalid: [
-        'CH 123456',
-        '12345',
+        // strictly valid
```

Comment:
I'm not sure what the opinion of the others is, but I like that you mentioned what is wrong with the tests

Reply:
Well, I wasn't sure if it might be too much commenting but I think it helps understanding the purpose of the test-cases.